src/main/scala/com/lightbend/rp/sbtreactiveapp/SbtReactiveApp.scala

```diff
@@ -33,7 +35,7 @@ object SbtReactiveApp {
     version: Option[String],
     secrets: Set[Secret],
     modules: Seq[(String, Boolean)],
-    akkaClusterBootstrapSystemName: Option[String]): Map[String, String] = {
+    akkaClusterBootstrapSystemName: Option[String]): Vector[(String, String)] = {
```

Comment:
Why Vector instead of Seq?  I'm just skimming along here but it looks like a Seq which used to be converted to Map, which would have the interesting side effect of de-duping keys.  Converting to Vector doesn't do anything interesting that I can see, so ... leave it as Seq maybe?  Or did I miss something?

Is it okay to assume there are no dupes, or have we lost something there?  If we still need to dedupe, but want preserved insert order, we could use a ListMap.

Reply:
I don't think duplicates are a concern here. If they were, just keeping the latest may not be what we want.

`++` produces a `mutable.Seq`, but the caller to `labels` expects it to be an `immutable.Seq`.

We need to choose some immutable Seq to convert to. I used Vector as the static and run-time types. At run-time it's probably not a bad choice versus List. In terms of static-type, would you prefer we upcast to immutable.Seq ?

Reply:
I think method signature of `immutable.Seq` is good, created via `.toVector`.